content/telegraf/v1.19/introduction/installation.md

```diff
@@ -46,7 +46,14 @@ aren't synchronized with NTP, the timestamps on the data can be inaccurate.
 {{% tab-content %}}
 Debian and Ubuntu users can install the latest stable version of Telegraf using the `apt-get` package manager.
 
-**Ubuntu:** Add the InfluxData repository with the following commands:
+**Ubuntu:** Telegraf is available in recent versions of Ubuntu.
```

Comment:
Do we know when specific versions of Ubuntu? Or at least the earliest supported version?